docs/enterprise/console-settings.yaml

```
@@ -0,0 +1,238 @@
+settings:
+  - name: "Reports"
+    settings:
+      - name: "Traffic"
+        doc: |
+          View the traffic running through Pomerium. Filter by [Route][route-concept] name, or date range.
+
+          ![The Traffic page in Pomerium Enterprise](../img/traffic-fullpage.png)
+      - name: "Runtime"
+        doc: |
+          Monitor how much system resources Pomerium is consuming. Filter by date range, service, and instance.
```

```suggestion
          Monitor how many system resources Pomerium is consuming. Filter by date range, service, and instance.
```

I'm not sure if this is grammar-ratical.

---

🤷

---

I think either are technically correct, but like how your change reads better. :+1: